autosklearn/automl.py

```diff
@@ -917,6 +944,144 @@ def refit(self, X, y):
         self._can_predict = True
         return self
 
+    def fit_pipeline(
+        self,
+        X: SUPPORTED_FEAT_TYPES,
+        y: SUPPORTED_TARGET_TYPES,
+        task: int,
```

Comment:
Shouldn't this accept the `feat_type`, too? (And all subclasses and the estimator as well)

Reply:
Yes, it is already there in [here](https://github.com/automl/auto-sklearn/blob/e5539dc9ba7c6c06d9b89aa05039093fc461de8d/autosklearn/automl.py#L1006) but I will add it to the signature